sites/partners/src/components/listings/PaperListingForm/sections/ApplicationTypes.tsx

```diff
@@ -38,7 +38,6 @@ const ApplicationTypes = ({ listing }: { listing: FormListing }) => {
   const { register, setValue, watch, errors } = useFormContext()
   // watch fields
   const digitalApplicationChoice = watch("digitalApplicationChoice")
-  const commonDigitalApplicationChoice = watch("commonDigitalApplicationChoice")
```

Comment:
can we add a check in backend/core for the PUT/POST (not sure which) that throws if `digitalApplicationChoice === YesNoAnswer.Yes` is set?

Reply:
possibly here? https://github.com/metrotranscom/Doorway/blob/main/backend/core/src/listings/listings.service.ts#L217

Reply:
Added the check. I imagine you meant commonDigitalApplicationChoice instead of digitalApplicationChoice since that one is still an option.